Date/MomentFormatConverter.php

```diff
@@ -59,6 +57,42 @@ class MomentFormatConverter
      */
     public function convert($format)
     {
-        return strtr($format, self::$formatConvertRules);
+        $size = strlen($format);
+        $foundOne = false;
+        $output = '';
+        //process the format string letter by letter
+        for ($i = 0; $i < $size; $i++) {
+            //if finds a '
+            if ($format[$i] == '\'') {
+                //if the next character are T' forming 'T', send a T to the output
+                if ($format[$i+1] == 'T' && $format[$i+2] == '\'') {
+                    $output = $output . 'T';
+                    $i = $i+2;
+                } else {
+                    //if it's no a 'T' then send whatever is inside the '' to the output, but send it inside [] (useful for cases like the brazilian translation that uses a 'de' in the date)
```

Comment:
Please wrap your comments so that the line is not longer than 80 chars (`gqq` in vim)